backend-agent/app/db/utils.py

```diff
@@ -0,0 +1,71 @@
+from .models import (
+    Attack as AttackDB,
+    db,
+    AttackModel as AttackModelDB,
+    AttackResult as AttackResultDB,
+    ModelAttackScore as ModelAttackScoreDB,
+)
+
+
+def save_to_db(attack_results):
+    """
+    Persist the SuiteResult into the database.
+    Returns a list of AttackResults that were added.
+    """
+    inserted_records = []
+
+    attack_name = attack_results.attack.lower()
+    success = attack_results.success
+    vulnerability_type = attack_results.vulnerability_type.lower()
+    details = attack_results.details  # JSON column
+    model_name = details.get('target_model').lower() if 'target_model' in details else 'unknown'
```

Comment:
if the target model name is unknown, shouldn't we just return without adding anything to the db?